### ISSUE TITLE
Rectangle select not working

### ISSUE THREAD

**Issue by matthewspeir** [maximilianh/cellBrowser#82]
It seems like the rectangle select doesn't function after one of the latest updates. 

Here's an example of a broken rectangle select, created 2019-03-11: https://genome-test.gi.ucsc.edu/~mspeir/cb_pancreas_loom/?ds=HCA_Pancreas_Aging_Loom

Here's an example with a working rectangle select, created 2019-02-26: https://genome-test.gi.ucsc.edu/~mspeir/cb_pancreas_mtx/?ds=HCA_Pancreas_mtx

**Comment by maximilianh** [comment]
No, I think this has to do with bundle_uuid being a string field with
unique values. It's possible that I've never had a case like this before...
I'll look into it. Usually the unique fields are the cell_id field and it's
always the first one. Maybe something here is different. Thanks!

**Comment by maximilianh** [comment]
Oh, now I understand, this makes complete sense: when you multi-select
cells, the left menu tries to summarize. It tries to show histograms
of the values that you selected. But with bundle_uuid this cannot
work, as it's a unique identifier. I now stopped doing this.

I've made a change which I hope works, I can't test it yet, since I
don't know where this data is.  Can you tell me where this data is on
dev?

**Comment by maximilianh** [comment]
I made a release 0.4.58 with what I think is a fix for this, but
haven't tested it yet.

**Comment by matthewspeir** [comment]
The starting data for https://genome-test.gi.ucsc.edu/~mspeir/cb_pancreas_loom/?ds=HCA_Pancreas_Aging_Loom is /hive/users/mspeir/cellbrowserTest/pancreas/new_metadata_test/matrix_files/Single_cell_transcriptome_analysis_of_human_pancreas.loom 

I'll update my cell browser version and test this out now.

**Comment by matthewspeir** [comment]
Hmm, I think something else sprung loose. Now the cells don't even show up. Looks like it's treating all fields as 'unique id' fields:

![cellbrowserUniqueIdField](https://user-images.githubusercontent.com/11509990/54294771-0f2ffb80-456f-11e9-9064-e8b044608675.png)


See this one: 
https://genome-test.gi.ucsc.edu/~mspeir/cb_pancreas_loom/?ds=HCA_Pancreas_Aging_Loom

or this one:
https://genome-test.gi.ucsc.edu/~mspeir/cb_pancreas_loom_v2/?ds=HCA_Pancreas_Aging_Loom

Data location:
```
/hive/users/mspeir/cellbrowserTest/pancreas/new_metadata_test/matrix_files/cbScanpyOut_pancreas_aging_loom_v2
```
Command run:
```
cbBuild -o ~/public_html/cb_pancreas_loom_v2
```

**Comment by maximilianh** [comment]
Yes, as I said, I couldn't really test this. I'll fix it. Thanks for
finding this, unique fields are not something I've tested a lot, they're
relatively rare.

**Comment by matthewspeir** [comment]
Looks like this is fixed in the latest (0.4.62). Closing
